.github/workflows/renovate.yaml

```diff
@@ -0,0 +1,15 @@
+name: Renovate
+on:
+  schedule:
+    - cron: '0/15 * * * *'
```

Comment:
4 times an hour is quite aggressive and might exhaust the free tier CI hours/month.
Any chance to reduce the frequency?